packages/nextcloud/test/core_test.dart

```diff
@@ -69,8 +69,10 @@ void main() {
       expect(navigationApps.ocs.data[5].id, 'news');
     });
 
+    /*
+    TODO: This test only works on 28+ due to a bug fix with the status
```

Comment:
I'd like to leave the test implemented and just skip it.

I hate commented tests as it's harder to see them from the UI of the IDE and I tend to forget that we even have it

Reply:
Ah I forgot that you can skip tests